src/Microsoft.AspNet.Mvc.Abstractions/ModelBinding/ModelMetadata.cs

```diff
@@ -290,6 +290,11 @@ public string PropertyName
         public abstract string TemplateHint { get; }
 
         /// <summary>
+        /// Gets a value that indicates whether children of the model should be validated.
```

Comment:
Not sure how often we use the term "child" but maybe clarifying whether it means sub-properties, collection items, etc.?